README.md

```diff
@@ -31,11 +31,14 @@ LOGGING = {
         'console': {
             'class': 'logging.StreamHandler',
         },
+        'database': {
+            'class': 'logging.'
+        },
     },
     'loggers': {
         'django.request': {
-            'handlers': ['console'],
-            'level': 'DEBUG',  # change debug level as appropiate
+            'handlers': ['console', 'database'],  # change handlers as appropriate
```

Comment:
Same as above, we only want to maintain a minimal example here

Reply:
Removed.